etna/auto/utils.py

```diff
@@ -4,6 +4,20 @@
 from hashlib import md5
 from typing import Any
 from typing import Callable
+from typing import Dict
+
+import optuna
+
+from etna.distributions import BaseDistribution
+from etna.distributions import CategoricalDistribution
+from etna.distributions import FloatDistribution
+from etna.distributions import IntDistribution
+
+_DISTRIBUTIONS = {
```

Comment:
Do these lambdas work in distributed env?

Reply:
I'll try to make smth more robust, because it is now dataclasses and it can be made much simpler.

Reply:
Fixed.